packages/css/README.md

```diff
@@ -1,28 +1,31 @@
-# emotion
+# @emotion/css
 
-The [emotion](https://www.npmjs.com/package/emotion) package is framework agnostic and the simplest way to use Emotion.
+The [@emotion/css](https://www.npmjs.com/package/@emotion/css) package is framework agnostic and the simplest way to use Emotion.
 
 ## Table of Contents
 
-- [Quick Start](#quick-start)
-- [API](#api)
-  - [Generate Class Names — `css`](#css)
-  - [Global Styles — `injectGlobal`](#global-styles)
-  - [Animation Keyframes — `keyframes`](#animation-keyframes)
-  - [Composing Class Names — `cx`](#cx)
-- [Server Side Rendering](/docs/ssr.mdx#api)
```

Comment:
SSR and babel plugin were worth mentioning here

Reply:
They're still there in the `@emotion/css` README

Reply:
I've just checked and it seems that they are gone, this is the whole TOC:
```md
- [@emotion/css](#emotioncss)
  - [Table of Contents](#table-of-contents)
  - [Quick Start](#quick-start)
  - [API](#api)
    - [css](#css)
      - [String Styles](#string-styles)
      - [Object Styles](#object-styles)
      - [Array of Object Styles](#array-of-object-styles)
    - [Global Styles](#global-styles)
    - [Animation Keyframes](#animation-keyframes)
    - [cx](#cx)
```